app/bundles/CampaignBundle/Config/services.php

```diff
@@ -37,6 +37,7 @@
     $services->alias('mautic.campaign.executioner.inactive', Mautic\CampaignBundle\Executioner\InactiveExecutioner::class);
     $services->alias('mautic.campaign.executioner.scheduled', Mautic\CampaignBundle\Executioner\ScheduledExecutioner::class);
     $services->alias('mautic.campaign.scheduler.optimized', Mautic\CampaignBundle\Executioner\Scheduler\Mode\Optimized::class);
+    $services->alias('mautic.campaign.event_logger', Mautic\CampaignBundle\Executioner\Logger\EventLogger::class);
```

Comment:
```suggestion
    $services->alias('mautic.campaign.event_logger', Mautic\CampaignBundle\Executioner\Logger\EventLogger::class);
    $services->alias('mautic.campaign.executioner.kickoff' Mautic\CampaignBundle\Executioner\KickoffExecutioner::class);
```

Reply:
@escopecz  I had already searched in complete project, `mautic.campaign.executioner.kickoff` was only defined and not being injected in any other class so i have not added alias for that.
still do you thing that alias should be created in this PR ?

Reply:
I'd create the aliases either way. Sometimes the service keys are being constructed dynamically so it's not easy to find all occurrences. And some plugins can rely on that. We can drop them in a bigger batch in some of the future major releases.